frontend/src/store/modules/place/actions.js

```diff
@@ -143,6 +143,96 @@ export default {
         });
     },
 
+    likePlaceReview: (context, data) => {
+        httpService.post(`reviews/${data.id}/like`)
+            .then( (res) => {
+                let review = context.state.places.find( (place) => { 
+                    return place.id === parseInt(data.placeId); 
+                }).review;
+                if (review.like === STATUS_NONE) {
+                    context.commit('SET_PLACE_REVIEW_LIKE_STATE', {
+                        placeId: data.placeId,
+                        likeState: STATUS_LIKED
+                    });
+                    context.commit('SET_PLACE_REVIEW_LIKE_COUNT', {
+                        placeId: data.placeId,
+                        count: review.likes + 1
+                    });
+                } else if (review.like === STATUS_LIKED) {
```

Comment:
All logic for 
if (review.like === STATUS_NONE) {...} 
else if (if (review.like === STATUS_LIKED) {) {...}
can be own action method, `updateReviewLikeStatus(review, placeId)`